response/writer.go

```diff
@@ -23,17 +23,17 @@ func WriteResponse(ctx context.Context, w http.ResponseWriter, babbageResponse *
 	}
 }
 
-func writeResponse(ctx context.Context, w http.ResponseWriter, babbageResponse *http.Response, additionalHeaders map[string]string) {
+func writeResponse(ctx context.Context, w http.ResponseWriter, babbageResponse *http.Response, newHeaders map[string]string) {
```

Comment:
`newHeaders` is not accurate enough, but at the same time, I don't know what else to suggest. They're not new headers, because they could already exist in the response. These are headers that we will definitely set in the response, overwriting existing ones. Can you think of a good name for this? If not, we should probably write some Godoc for this function, and explain it there.

Reply:
I've gone with `overrideHeaders` without any documentation explaining the purpose, happy to add some though if you feel its needed

Reply:
`overrideHeaders` is a good, descriptive name 👍